debug.log

```diff
@@ -1,2 +1,10 @@
 [1021/234833.573:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```

Comment:
Why do you have a log file in the pull request, it is better to totally remove it and put into `gitignore`.

Reply:
Thanks
It's already done